frontend/src/pages/RoadmapGraphPage.tsx

```diff
@@ -61,79 +87,74 @@ export const RoadmapGraphPage = () => {
   }, [roadmapsVersions, selectedVersion]);
 
   const numFormat = new Intl.NumberFormat(undefined, {
     minimumFractionDigits: 0,
     maximumFractionDigits: 1,
   });
 
+  const dimensions = ({ work, value }: VersionWorkAndValues) => ({
+    width: work,
+    height: value,
+  });
+
+  const limits = {
+    minWidth: 100,
+    maxWidth: 300,
+    minHeight: 100,
+    maxHeight: 300,
+  };
+
   return (
     <div className={classes(css.plannerPagecontainer)}>
-      <div className={classes(css.graphOuter)}>
-        <div className={classes(css.titleContainer)}>
-          <h2 className={classes(css.graphTitle)}>{t('workValueTitle')}</h2>
-          <InfoTooltip title={t('tooltipMessage')}>
-            <InfoIcon className={classes(css.tooltipIcon, css.infoIcon)} />
-          </InfoTooltip>
-        </div>
-
-        <div className={classes(css.graphInner)}>
-          <div className={classes(css.graphItems)}>
-            {versions?.map((ver) => {
-              const numTasks = ver.tasks.length;
-              const w = Math.max(100, 60 * (ver.work / 5));
-              const h = Math.min(350, Math.max(100, 45 * Math.log2(ver.value)));
-              return (
-                <div
-                  className={classes(css.graphItem, {
-                    [css.selected]: ver.id === selectedVersion?.id,
-                  })}
-                  style={{ width: `${w}px`, height: `${h}px` }}
-                  key={ver.id}
-                  onClick={() => setSelectedVersion(ver)}
-                  onKeyPress={() => setSelectedVersion(ver)}
-                  role="button"
-                  tabIndex={0}
-                >
-                  <div className={classes(css.versionData)}>
-                    <div className={classes(css.ratingDiv)}>
-                      <BusinessIcon size="xxsmall" color={colors.azure} />
-                      <p>{numFormat.format(ver.value)}</p>
-                    </div>
-                    <div className={classes(css.ratingDiv)}>
-                      <WorkRoundIcon size="xxsmall" color={colors.azure} />
-                      <p>{numFormat.format(ver.work)}</p>
-                    </div>
-                    <div className={classes(css.dash)} />
-                    <div className={classes(css.ratingDiv)}>
-                      <ListIcon />
-                      <p>{numTasks}</p>
-                    </div>
-                  </div>
-                  <p
-                    className={classes(css.versionTitle, {
-                      [css.selected]: ver.id === selectedVersion?.id,
-                    })}
-                  >
-                    {ver.name}
-                  </p>
-                </div>
-              );
-            })}
+      <BlockGraph
+        title={
+          <div className={classes(css.titleContainer)}>
+            <h2 className={classes(css.graphTitle)}>{t('workValueTitle')}</h2>
+            <InfoTooltip title={t('tooltipMessage')}>
```

Comment:
I think we should adding these tooltips messages soon, it could be a good to have them before the demo gets out. Our designer might have good ideas for the contents of these.